chain/chain_config.go

```diff
@@ -25,13 +25,26 @@ import (
 	"github.com/ledgerwatch/erigon/zk/zkchainconfig"
 )
 
+type ForkId uint64
+
 const (
-	ForkID5Dragonfruit = 5
-	ForkID6IncaBerry   = 6
-	ForkID7Etrog       = 7
-	ForkID88Elderberry = 8
+	ForkID4            ForkId = 4
+	ForkID5Dragonfruit ForkId = 5
+	ForkID6IncaBerry   ForkId = 6
+	ForkID7Etrog       ForkId = 7
+	ForkID88Elderberry ForkId = 8
```

Comment:
There is forkid9 elderberry2 with no changes for us, but better if we have it here as well